dockerfile-image-update-cmdtool/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGithubUtil.java

```diff
@@ -194,20 +192,24 @@ public void updateStore(String store, String img, String tag) throws IOException
             String ownerOrg = myself.getLogin();
             storeRepo = githubUtil.getRepo(Paths.get(ownerOrg, store).toString());
         } catch (IOException e) {
-            storeRepo = githubUtil.createPrivateRepo(store);
+            storeRepo = githubUtil.createPublicRepo(store);
         }
-        updateStoreOnGithub(storeRepo, STORE_JSON_FILE, img, tag);
+        updateStoreOnGithub(storeRepo, Constants.STORE_JSON_FILE, img, tag);
     }
 
     protected void updateStoreOnGithub(GHRepository repo, String path, String img, String tag) throws IOException {
-        GHContent content;
         try {
-            content = repo.getFileContent(path);
+            repo.getFileContent(path);
         } catch (IOException e) {
-            GHContentUpdateResponse contentCreate = repo.createContent("", "initializing store", path);
-            content = contentCreate.getContent();
+            repo.createContent("", "initializing store", path);
+
         }
-        if (content.getType().equals(GITHUB_FILE)) {
+
+        String latestCommit = repo.getBranches().get(repo.getDefaultBranch()).getSHA1();
+        log.info("Loading image store at commit {}", latestCommit);
+        GHContent content = repo.getFileContent(path, latestCommit);
+
+        if (content.getType().equals(Constants.GITHUB_FILE)) {
```

Comment:
Use `content.isFile()`. Once done, no need for `GITHUB_FILE`

Reply:
See comment above